go.mod

```diff
@@ -112,3 +112,7 @@ require (
 )
 
 go 1.14
+
+// Only works with: GOPROXY=direct (https://github.com/golang/go/issues/32955#issuecomment-509318558)
+// Still, the branch name will get rewritten with the commit..
+replace github.com/ipfs/go-bitswap => github.com/ipfs/go-bitswap monitor/data-stall
```

Comment:
It's usually simpler (especially when giving code to others) to just specify the commit directly without a replace directive.

1. Specify `github.com/ipfs/go-bitswap@monitor/data-stall` in the dependencies.
2. Run `GOPROXY=direct go mod tidy` (will resolve `monitor/data-stall` to a commit.

After that, everything will work even with the default GOPROXY.

Reply:
(not important, just FYI)

Reply:
Yes, this is not meant for actual production, I just wanted a way to link both development branches without having to manually update on each change. This was the only way I found to link to a moving target.